src/main/java/com/juniorstart/juniorstart/security/oauth2/CustomOAuth2UserService.java

```diff
@@ -19,16 +19,27 @@
 
 import java.util.Optional;
 
+/** Represents a OAuth2 user service.
+ * @author Grzegorz Szczęsny
+ * @version 1.0
+ * @since 1.0
+ */
 @Service
 public class CustomOAuth2UserService extends DefaultOAuth2UserService {
 
     private UserDao userDao;
```

Comment:
Tutaj to samo. UserService powinno być zamiast UserDao. Jak brakuje jakieś metody to sobie dorobić

Reply:
Spoko